src/main/java/com/cleanroommc/groovyscript/compat/loot/Loot.java

```diff
@@ -0,0 +1,225 @@
+package com.cleanroommc.groovyscript.compat.loot;
+
+import com.cleanroommc.groovyscript.api.GroovyBlacklist;
+import com.cleanroommc.groovyscript.api.GroovyLog;
+import com.cleanroommc.groovyscript.core.mixin.MinecraftServerAccessor;
+import com.cleanroommc.groovyscript.core.mixin.loot.LootPoolAccessor;
+import com.cleanroommc.groovyscript.core.mixin.loot.LootTableAccessor;
+import groovy.lang.Closure;
+import net.minecraft.client.Minecraft;
+import net.minecraft.inventory.EntityEquipmentSlot;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.WorldServer;
+import net.minecraft.world.storage.loot.*;
+import net.minecraft.world.storage.loot.conditions.KilledByPlayer;
+import net.minecraft.world.storage.loot.conditions.LootCondition;
+import net.minecraft.world.storage.loot.conditions.RandomChance;
+import net.minecraft.world.storage.loot.conditions.RandomChanceWithLooting;
+import org.jetbrains.annotations.ApiStatus;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Random;
+import java.util.concurrent.atomic.AtomicReference;
+
+public class Loot {
+
+    @GroovyBlacklist
+    @ApiStatus.Internal
+    public void onReload() {
+        TABLES.clear();
+        TABLE_MANAGER = new LootTableManager(null);
+    }
+
+    @GroovyBlacklist
+    @ApiStatus.Internal
+    public void afterScriptRun() {
+        if (Minecraft.getMinecraft().isIntegratedServerRunning() && Minecraft.getMinecraft().getIntegratedServer() != null) {
+            for (WorldServer world : ((MinecraftServerAccessor) Minecraft.getMinecraft().getIntegratedServer()).getWorlds()) {
+                world.getLootTableManager().reloadLootTables();
+            }
+        }
+    }
+
+    public static final Map<ResourceLocation, LootTable> TABLES = new HashMap<>();
+    public static LootTableManager TABLE_MANAGER;
+
+    public LootTable getTable(String name) {
+        AtomicReference<LootTable> val = new AtomicReference<>(null);
+        TABLES.forEach( (rl, lt) -> {
+            if (rl.toString().equals(name)) {
+                val.set(lt);
+            }
+        } );
+        if (val.get() == null) GroovyLog.msg("GroovyScript found 0 LootTable(s) named " + name).post();
+        return val.get();
+    }
+
+    public void removeTable(String name) {
+        TABLES.forEach( (rl, lt) -> {
+            if (rl.toString().equals(name)) {
+                lt = LootTable.EMPTY_LOOT_TABLE;
+            }
+        } );
+    }
```

Comment:
```java
public void removeTable(String name) {
    TABLES.remove(new ResourceLocation(name));
}
```

Reply:
This won't work since the event handler does removing by swapping the to-be-loaded table with an empty table, so long as the resourcelocation is present in the TABLES map. I can make the code more concise with just a TABLES.put(rl, EMPTY_TABLE) though